tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Product Card UI Design</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
```

Comment:
Always link `CSS` file in last.

Reply:
No, Why would you do that. ?

Reply:
Sorry, I've been mistaken.

Reply:
I taught loading CSS first will load it in default font, and then apply the custom font,

Reply:
I taught loading CSS first will load it in default font, and then apply the custom font.

Reply:
Yes Done. @shaikh-7abish